File: 프로그래머스/0/120822. 뒤집힌 문자열/뒤집힌 문자열.java
class Solution {
    public String solution(String my_string) {
        
        char []arr = new char[my_string.length()];
        for (int i = 0; i < my_string.length(); i++) {
            arr[i] = my_string.charAt(i);
        }
        
        String answer = "";
        for (int i = my_string.length()-1; i >= 0; i--) {
            answer = answer + arr[i];
        }
        
        return answer;
    }
}

File: 프로그래머스/0/120822. 뒤집힌 문자열/뒤집힌 문자열.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>
#include <string.h>

char* solution(const char* my_string) {
    char* answer = (char*)malloc(strlen(my_string)*sizeof(char));
    
    int count = 0; 
    for (int i = strlen(my_string)-1; i >= 0; i--) {
        answer[count] = my_string[i];
        count++;
    }
    
    answer[strlen(my_string)] = '\0';
    
    return answer;
}